packages/manager/apps/pci-private-network/src/interface/index.ts

```diff
@@ -0,0 +1,15 @@
+export interface ResponseAPIError {
```

Comment:
Could you please provide more information about the type name? I'm not sure if it's the same as `ResponseAPIError` because I couldn't find it. Alternatively, could you please provide the alias import from core if you don't mind?

Reply:
Its definition is located in `packages/manager/core/api/src/client.ts`

and you can import it like this : `import {ApiError} from '@ovh-ux/manager-core-api';`
It just misses the headers definition I think but you can improve the type and add them if you want.

Currently it's defined like this :
```
export type ApiError = AxiosError<{ message: string }>;
```

but we can add the headers like this for example : 
```
export type ApiError = AxiosError<{ message: string }> & {
  response: { headers: Record<string, string> & { 'x-ohv-queryid': string } };
};
```